HW6/KaterynaYusypenko/Task1.py

```diff
@@ -0,0 +1,15 @@
+even_numbers = []
+odd_numbers_divisible_by_3 = []
+other_numbers = []
+
+for item in range(1, 11):
+    if item % 2 == 0:
+        even_numbers.append(item)
+    if item % 3 == 0:
```

Comment:
according to the conditions of the problem, it is necessary to adjust this condition to the following
if item % 3 == 0 and item % 2 == 1: